examples/limitrange/README.md

```diff
@@ -1,4 +1,147 @@
-Please refer to this [doc](https://github.com/GoogleCloudPlatform/kubernetes/blob/620af168920b773ade28e27211ad684903a1db21/docs/design/admission_control_limit_range.md#kubectl).
+Limit Range
+========================================
+By default, a pod will run with unbounded CPU and memory limits.  This means that any pod in the
+system will be able to consume as much CPU and memory on the node that executes the pod.
+
+Users may want to impose restrictions on the amount of resource a single pod in the system may consume
+for a variety of reasons.
+
+For example:
+
+1. Each node in the cluster has 2GB of memory.  The cluster operator may want to reject any pod that requests
+more than that amount prior to admitting the pod in the cluster since that pod could never be scheduled by the
+system.  As a result, an operator has a desire to set the maximum amount of resource a single pod could consume
+on a system in order to increase the odds that a pod could fit on a node in the cluster.
```

Comment:
nit: This is not technically accurate, right?  Just adding the limit does not affect schedulability?  It just improves error reporting.  Worth clarifying?